server/routes/bookedtoday/arrivals/searchforexisting/singleExistingRecordFoundController.test.ts

```diff
@@ -5,14 +5,26 @@ import { appWithAllRoutes, stubCookies } from '../../../__testutils/appSetup'
 import Role from '../../../../authentication/role'
 import config from '../../../../config'
 import { State } from '../state'
+import { createPotentialMatch } from '../../../../data/__testutils/testObjects'
+import { createMockExpectedArrivalsService } from '../../../../services/__testutils/mocks'
+
+const potentialMatch = createPotentialMatch({
+  firstName: 'Jim',
+  lastName: 'Smith',
+  dateOfBirth: '1973-01-08',
+  prisonNumber: 'A1234AB',
+  pncNumber: '01/98644M',
+})
+const expectedArrivalsService = createMockExpectedArrivalsService()
 
 let app: Express
 
 beforeEach(() => {
   config.confirmNoIdentifiersEnabled = true
-  app = appWithAllRoutes({ roles: [Role.PRISON_RECEPTION] })
-
+  expectedArrivalsService.getMatchingRecords.mockResolvedValue([potentialMatch])
+  app = appWithAllRoutes({ services: { expectedArrivalsService }, roles: [Role.PRISON_RECEPTION] })
   stubCookies([
+    // FIXME: Look into why removing the newArrival state breaks the tests
```

Comment:
Not entirely sure why removing the new arrival state cookie breaks the tests. Finding it hard to debug as well 🤔